infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/runtime/LogReplicationClientRouter.java

```diff
@@ -280,7 +286,8 @@ public void sendMessage(CorfuMsg message) {
         if (runtimeFSM.getRemoteLeader().isPresent()) {
             String remoteLeader = runtimeFSM.getRemoteLeader().get();
             this.requestSample = MeterRegistryProvider.getInstance().map(Timer::start);
-            channelAdapter.send(remoteLeader, CorfuMessageConverterUtils.toProtoBuf(message));
+            // TODO(vbrajkovic)
+            //channelAdapter.send(remoteLeader, CorfuMessageConverterUtils.toProtoBuf(message));
```

Comment:
Doesn't this need to be uncommented? or the complete method removed if deprecated?

Reply:
This function is no longer used and it will be removed in the subsequent PR. I needed to comment it out since `CorfuMessageConverterUtils` has been removed.